olp-cpp-sdk-dataservice-read/src/CatalogClientImpl.h

```diff
@@ -61,10 +61,18 @@ class CatalogClientImpl final {
   client::CancellableFuture<CatalogResponse> GetCatalog(
       const CatalogRequest& request);
 
+  client::CancellationToken GetLatestVersion(CatalogVersionRequest request,
+                                             CatalogVersionCallback callback);
+
+  client::CancellableFuture<CatalogVersionResponse> GetLatestVersion(
+      CatalogVersionRequest request);
+
+  OLP_SDK_DEPRECATED("Will be removed in 1.0")
   client::CancellationToken GetCatalogMetadataVersion(
       const CatalogVersionRequest& request,
       const CatalogVersionCallback& callback);
 
+  OLP_SDK_DEPRECATED("Will be removed in 1.0")
```

Comment:
we can just rename methods, no need to mark private code as deprecated

Reply:
sure thing!